src/ui/modules/ProductComparison.tsx

```
@@ -0,0 +1,100 @@
+              ))}
+            </tr>
+          </thead>
+          <tbody>
+            {features?.map((feature, ix) => (
+              <tr className="border-b" key={feature._key || ix}>
+                <td className="p-4 font-medium">{feature.name}</td>
+                {feature.featureDetails?.map((featureDetail, idx) => (
+                  <td
+                    key={idx}
+                    className={`p-4 text-center font-semibold ${
+                      products && products[idx]?.highlight 
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)